flink-external-resources/flink-external-resource-gpu/src/test/resources/test-coordination-mode.sh

```diff
@@ -34,39 +43,36 @@ test_privilege_mode() {
       fi
     done
   done
-
-  echo ${output1[1]} ${output1[0]}
-  echo ${output2[1]} ${output2[0]}
-  clean_state
 }
 
-test_check_dead() {
-  echo "0" >> /var/tmp/flink-gpu-assignment
-  IFS=',' read -r -a output1 <<< $(bash -c "$(dirname "$0")/testing-gpu-discovery.sh 2 --privilege --check-dead")
-  IFS=',' read -r -a output2 <<< $(bash -c "$(dirname "$0")/testing-gpu-discovery.sh 2 --privilege --check-dead")
+test_preempt_from_dead_processes() {
+  test_pid=$(shuf -i 1-32768 -n 1)
+  while [ $(ps -p $owner | grep -c $owner) -ne 0 ]
+  do
+    test_pid=$(shuf -i 1-32768 -n 1)
+  done
```

Comment:
`test_pid=$(shuf -i 1-32768 -n 1)` can be deduplicated with the following approach.
```
local test_pid
while [[ -z $test_pid || $(ps -p $owner | grep -c $owner) -ne 0 ]]
do
    test_pid=$(shuf -i 1-32768 -n 1)
done
```